src/addressResolvers/ens.ts

```diff
@@ -25,33 +26,44 @@ function normalizeHandles(names: string[]) {
   });
 }
 
+function normalizeAddresses(addresses: Address[]): Address[] {
+  return addresses.filter(isEvmAddress);
+}
+
+function normalizeHandles(names: Handle[]): Handle[] {
+  return normalizeEns(names).filter(h => h);
+}
```

Comment:
why seperate function to just filter? we can filter in `normalizeEns` itself correct?

Reply:
In lens resolver, both `map` and `filter` are done in same function `normalizeHandles`, we can do same here

Reply:
`normalizeHandles` and `normalizeAddresses` have the same signature everywhere.

`normalizeEns` is just for ENS, and is also used in another case, to detect variable name, without filtering out invalid names. Merging everything was the cause of the previous issue last week, about wrong ENS assosiated to wrong address.